administrator/language/de-DE/plg_user_joomla.ini

```diff
@@ -8,12 +8,13 @@ PLG_USER_JOOMLA="Benutzer – Joomla!"
 PLG_USER_JOOMLA_FIELD_AUTOREGISTER_LABEL="Automatisch Benutzer erstellen"
 PLG_USER_JOOMLA_FIELD_FORCELOGOUT_LABEL="Alle Sitzungen beenden?"
 PLG_USER_JOOMLA_FIELD_MAILTOUSER_LABEL="Benachrichtigungs-E-Mail an den Benutzer"
-PLG_USER_JOOMLA_FIELD_STRONG_PASSWORDS_LABEL="Starke Passwörter"
 PLG_USER_JOOMLA_MAIL_MAIL_DESC="Diese E-Mail wird an einen neuen Benutzer gesendet, der gerade im Backend angelegt wurde."
 PLG_USER_JOOMLA_MAIL_MAIL_TITLE="Benutzer: Neuer Benutzer"
 PLG_USER_JOOMLA_NEW_USER_EMAIL_BODY="Hallo {NAME},\n\n\ndiese E-Mail-Adresse wurde von einem Administrator auf der Seite {SITENAME} für einen neuen Benutzer registriert.\n\nEs folgen Benutzername und Passwort zur Anmeldung auf der Seite {URL}\n\nBenutzername: {USERNAME}\nPasswort: {PASSWORD}\n\n\nBitte auf diese Nachricht nicht antworten, da sie automatisch vom System erstellt wurde und nur zur Information dient."
 PLG_USER_JOOMLA_NEW_USER_EMAIL_SUBJECT="Details zum neuen Benutzer"
+PLG_USER_JOOMLA_XML_DESCRIPTION="Steuert die Joomla!-Standard-Benutzer-Synchronisation.<br /><br /><strong>Warnung:</strong><br />Es muss mindestens ein Plugin aktiviert sein, das die Benutzersitzungen verwaltet, da man sonst jede Zugriffsmöglichkeit auf die Website verliert."
```

Comment:
```suggestion
PLG_USER_JOOMLA_XML_DESCRIPTION="Steuert die Joomla-Standard-Benutzer-Synchronisation.<br /><br /><strong>Warnung:</strong><br />Es muss mindestens ein Plugin aktiviert sein, das die Benutzersitzungen verwaltet, da man sonst jede Zugriffsmöglichkeit auf die Website verliert."
```

Reply:
@tecpromotion Auch in plg_user_joomla.sys.ini anpassen?

Reply:
Ja bitte. In der Schreibweise kein ! Verwenden. 
Haben wir vermutlich noch öfter so.